### DISCUSSION TITLE
main admin is not being logged (for spoke we want)

### DISCUSSION THREAD

**Discussion by shamsail** [mozilla/hubs#4531] [General Questions & Discussion]
Hi, I want to log into the account of the hubs cloud admin. I have main admin account (through which I have deployed the clouds hub on ec2 aws server) and also some more admin accounts. Previously , main admin as well as other admin accounts were working on login. but now main admin account is not being logged in but other admin accounts are being logged in.
Case scenario: when I'm putting email while login on hubs cloud, I'm getting an email with the magic link, but when I'll click the magic link , it also redirect to my hubs cloud website, but it's not verifying the main admin. I want that main admin can be verified because I have to use the spoke (because it only works with main admin). This thing was working before but not now.
Is this a issue? or something has changed? because I did not change anything from the code

**Comment by pattersonbl2** [comment]
Is it possible for you to attach a screenshot or a screen recording of the issue? I'm not fully understanding the issue with the summary above.  Are you unable to log into the Main admin account because the redirect link in the email doesn't verify or takes you to the website already signed in?

**Reply by shamsail** [reply]
The first one.. "email doesn't verify" for the main admin. 
you can check the attach images for the previous working. And now it's just redirecting and not verifying the email. That's why I can't login.
![IMG_20210817_174945](https://user-images.githubusercontent.com/30554980/129729862-236ad9f8-6cc3-472d-b06b-d0932bcece72.jpg)
![IMG_20210817_174959](https://user-images.githubusercontent.com/30554980/129729866-3214fd1a-12d2-4378-9fc4-bc27bf79f3f7.jpg)

**Reply by pattersonbl2** [reply]
> The first one.. "email doesn't verify" for the main admin. 
> 
> you can check the attach images for the previous working. And now it's just redirecting and not verifying the email. That's why I can't login.
> 
> ![IMG_20210817_174945](https://user-images.githubusercontent.com/30554980/129729862-236ad9f8-6cc3-472d-b06b-d0932bcece72.jpg)
> 
> ![IMG_20210817_174959](https://user-images.githubusercontent.com/30554980/129729866-3214fd1a-12d2-4378-9fc4-bc27bf79f3f7.jpg)
> 
> 

Before we terminate the instance, have you tried to clear the cache or a new browser. Sometimes the browser can get stuck with stale info causing issues.

**Comment by shamsail** [comment]
I have tried this already like to clear the cache or cookies.. even I have
tried on different browsers and different browser..
One thing , I'll mention it here I'm accessing the main admin account
outside of the aws region where I have deployed..
Because one of my colleague is accessing the main admin account from the
same region but I can't access. Is this a problem related to the region?
Because if it's so then why it was working before?  These are some thoughts
on this

**Reply by pattersonbl2** [reply]
This shouldn't be the case but I am not sure. You could try terminating the EC2 instance which will load a new one up to see if that fixes things

**Reply by pattersonbl2** [reply]
@shamsail  did you ever figure this issue out ?
